src/main/java/com/blamejared/compat/botania/commands/BotaniaBrewLogger.java

```diff
@@ -0,0 +1,30 @@
+package com.blamejared.compat.botania.commands;
+
+import crafttweaker.CraftTweakerAPI;
+import crafttweaker.mc1120.commands.CraftTweakerCommand;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.text.TextComponentString;
+import vazkii.botania.api.BotaniaAPI;
+
+public class BotaniaBrewLogger extends CraftTweakerCommand {
+
+	public BotaniaBrewLogger() {
+		super("botbrews");
+	}
+
+	@Override
+	public void executeCommand(MinecraftServer server, ICommandSender sender, String[] arguments) {
+		for (String key : BotaniaAPI.brewMap.keySet()) {
+			CraftTweakerAPI.logCommand(key);
+		}
+
+		if (sender != null) {
+			sender.sendMessage(new TextComponentString("List generated; see crafttweaker.log in your minecraft dir"));
```

Comment:
I'd recommend using something like ` sender.sendMessage(SpecialMessagesChat.getLinkToCraftTweakerLog("List generated;", sender));` to get a clickable directlink to the crafttweaker.log